safesens/account/schema.py

```diff
@@ -65,10 +65,18 @@ def mutate_and_get_payload(cls, root, info, **input):
 
 class AccountQuery(graphene.ObjectType):
     own_users = DjangoFilterConnectionField(UserNode, filterset_class=UserFilter)
+    get_technicians = DjangoFilterConnectionField(UserNode, filterset_class=UserFilter)
+    ###get all the users that are technicians
+    #so get me users where i am the manager
 
-    def resolve_own_users(self, info, **kwargs):
+    def resolve_get_technicians(self, info, **kwargs):
         current_user = info.context.user
+        if current_user.is_authenticated:
```

Comment:
This looks good. Please add the logic for when the user is not authenticated (raise GraphQLError)